src/js/multiDomainFirstParties.js

```diff
@@ -1343,6 +1369,20 @@ var multiDomainFirstPartiesArray = [
     "s-nbcnews.com",
   ],
   ["nefcuonline.com", "nefcu.com"],
+  [
+    "netease.com",
+
+    "126.com",
+    "126.net",
+    "127.net",
+    "163.com",
+
+    "blizzard.cn",
```

Comment:
Hello and thank you for the PR! I think we should remove `blizzard.cn` from the list for NetEase. Other than that, looks good!

Reply:
Hi!

I am not familiar with the detailed criteria. 

But (www.)blizzard.cn redirects to www.battlenet.com.cn, which is generally known to be run by NetEase. All (or most) of sub-sites of blizzard.cn clearly indicates that it is run by NetEase or affiliated / directly associated companies on the page footer. Besides, some sub-sites of blizzard.cn do have resources hosted NetEase's CDN and fail to load after the Badger learns. For example, http://hs.blizzard.cn refers to resources on 163.com and 127.net. 

Do you still think it is better to remove blizzard.cn?

Reply:
It seems that blizzard.cn and other service run by NetEase are independent enough and there is no data exchange between them via third-party cookies. So maybe we should remove blizzard.cn from NetEase's MDFP and add some of domains to "yellowlist"?

Reply:
MDFP is meant to group sites that belong to the same corporate entity so that Privacy Badger knows to treat them as first parties with respect to each other.

www.blizzard.cn redirects me to `https://www.battlenet.com.cn/zh/`.

www.battlenet.com.cn redirects to `https://www.blizzardgames.cn/zh/`.

These seem to be owned by Blizzard Entertainment, not NetEase.

I do see the problem with a page like http://hs.blizzard.cn/landing, where Blizzard and NetEase have some sort of partnership.

Reply:
>So maybe we should remove blizzard.cn from NetEase's MDFP and add some of NetEase's domains to "yellowlist"?

Sounds good to me. Could you open a new PR for the yellowlist update?

Reply:
Another way to look at what should and shouldn't be in the same MDFP group maybe is to consider whether the domain belongs with the other domains in the same Subject Alt Name field. I mean, assuming it's not one of those SSL certificates from a hosting/CDN company that has all sorts of unrelated domains in the SAN.